src/notebook/handler.ts

```diff
@@ -47,18 +47,64 @@ import {
   NotebookPanel
 } from './widget';
 
+import {
+  ISignal, Signal, clearSignalData
+} from 'phosphor-signaling';
+
+
+export
+class NotebookContext {
```

Comment:
Definitely

On Mon, Apr 18, 2016, 19:25 Steven Silvester <notifications@github.com>
wrote:

> In src/notebook/handler.ts
> <https://github.com/jupyter/jupyter-js-notebook/pull/184#discussion_r60151799>
> :
>
> > @@ -47,18 +47,64 @@ import {
> >    NotebookPanel
> >  } from './widget';
> >
> > +import {
> > +  ISignal, Signal, clearSignalData
> > +} from 'phosphor-signaling';
> > +
> > +
> > +export
> > +class NotebookContext {
>
> I'd like to chat about this tomorrow if we could.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/jupyter-js-notebook/pull/184/files/40917e4f38aba9cad4dd521375e400d157496119#r60151799>
>